src/lib/mongodb.ts

```diff
@@ -0,0 +1,28 @@
+import mongoose from 'mongoose';
+
+const MONGODB_URI = process.env.MONGODB_URI!;
+
+if (!MONGODB_URI) {
+  throw new Error('MONGODB_URI not included in .env.local');
```

Comment:
when this is executed on the service side environmental variables are not coming from .env.local so this message is wrong. Reword it to `MONGODB_URI environmental variable is not defined`